lib/perl/Genome/Model/Tools/Picard.pm

```diff
@@ -104,12 +131,12 @@ sub help_brief {
 sub help_synopsis {
     my $self = shift;
     return <<"EOS"
-gmt picard ...    
+gmt picard ...
 EOS
 }
 
-sub help_detail {                           
-    return <<EOS 
+sub help_detail {
+    return <<EOS
 More information about the Picard suite of tools can be found at http://picard.sourceforge.net/.
```

Comment:
Actually, the heuristic works for all but 4 of the existing commands, I'll take a crack at it now.